qiskit/dagcircuit/dagcircuit.py

```diff
@@ -1248,7 +1244,7 @@ def properties(self):
         """Return a dictionary of circuit properties."""
         summary = {"size": self.size(),
                    "depth": self.depth(),
-                   "width": self.width(),
+                   "width": self.num_qubits(),
```

Comment:
```suggestion
                   "width": self.width(),
                   "qubits": self.num_qubits(),
```
We should keep the meaning of `width` consistent throughout. Is there downstream code that depends on `properties['width']` being `num_qubits`? (I looked briefly within the transpiler passes, but didn't see any.)

Reply:
As long as any downstream consumer understands that the semantics of `self.width()` have changed, i.e., they were not depending on this being identical to `self.num_qubits` which is what `"width"` has stood for up to now. Make the change now.

Reply:
Change pushed